examples/CMakeLists.txt

```diff
@@ -1,30 +1,36 @@
 
-# add the examples subdirectories
-if(SFML_BUILD_NETWORK)
-    add_subdirectory(ftp)
-    add_subdirectory(sockets)
-endif()
-if(SFML_BUILD_NETWORK AND SFML_BUILD_AUDIO)
-    add_subdirectory(voip)
-endif()
-if(SFML_BUILD_AUDIO)
-    add_subdirectory(sound)
-    add_subdirectory(sound_capture)
-endif()
-if(SFML_BUILD_WINDOW)
-    add_subdirectory(window)
-endif()
-if(SFML_BUILD_GRAPHICS)
-    add_subdirectory(opengl)
-    add_subdirectory(shader)
-    if(SFML_OS_WINDOWS)
-        add_subdirectory(win32)
-    elseif(SFML_OS_LINUX OR SFML_OS_FREEBSD)
-        add_subdirectory(X11)
-    elseif(SFML_OS_MACOSX)
-        add_subdirectory(cocoa)
+# iOS Demo
+if(SFML_OS_IOS)
+    add_subdirectory(iOS)
+else(SFML_OS_IOS)
+
+    # add the examples subdirectories
+    if(SFML_BUILD_NETWORK)
+        add_subdirectory(ftp)
+        add_subdirectory(sockets)
+    endif()
+    if(SFML_BUILD_NETWORK AND SFML_BUILD_AUDIO)
+        add_subdirectory(voip)
+    endif()
+    if(SFML_BUILD_AUDIO)
+        add_subdirectory(sound)
+        add_subdirectory(sound_capture)
+    endif()
+    if(SFML_BUILD_WINDOW)
+        add_subdirectory(window)
+    endif()
+    if(SFML_BUILD_GRAPHICS)
+        add_subdirectory(opengl)
+        add_subdirectory(shader)
```

Comment:
Is the following if else blocks really needed? I mean SFML_OS_IOS and SFML_OS_WINDOWS can't both be set.